nebullvm/inference_learners/tensor_rt.py

```diff
@@ -248,6 +247,59 @@ def load(cls, path: Union[Path, str], **kwargs):
         )
 
 
+class PytorchTensorRTInferenceLearner(PytorchBaseInferenceLearner):
+    MODEL_NAME = "model_optimized.pt"
+
+    def __init__(
+        self, torch_model: torch.jit.ScriptModule, dtype: torch.dtype, **kwargs
```

Comment:
Does TensorRT need a TorchScript module as input?

Reply:
Yep, the model returned after compiling with torch_tensorrt is in TorchScript format